PyMca5/PyMcaGui/io/TiledWorker.py

```diff
@@ -0,0 +1,30 @@
+from PyQt5.QtCore import pyqtSignal, QObject, QRunnable
+
+
+class TiledWorkerSignals(QObject):
+    finished = pyqtSignal()
+    results = pyqtSignal(object)
+
+
+class TiledWorker(QRunnable):
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.signals = TiledWorkerSignals()
+        self.rows_per_page = kwargs["rows_per_page"]
+        self.current_page = kwargs["current_page"]
+        self.client = kwargs["client"]
+        self.search_results = kwargs["search_results"]
+        self.node_path_parts = kwargs["node_path_parts"]
```

Comment:
Do normal python keyword args not work with `QRunnable`?

That is, why not this?
```suggestion
    def __init__(
        self,
        *,
        client,
        current_page,
        node_path_parts,
        rows_per_page,
        search_results,
        **kwargs,
    ):
        super().__init__(**kwargs)
        self.signals = TiledWorkerSignals()
        self.rows_per_page = rows_per_page
        self.current_page = current_page
        self.client = client
        self.search_results = search_results
        self.node_path_parts = node_path_parts
```